appengine/standard/firebase/firetactoe/templates/_firebase_config.html

```diff
@@ -1,3 +1,3 @@
 REPLACE ME WITH YOUR FIREBASE WEBAPP CODE SNIPPET:
 
-https://console.firebase.google.com/project/_/overview
+https://console.firebase.google.com/project/_/overview
```

Comment:
could you just revert this file?

    git checkout a76c3dc7437b53bbc4b4c5a4e3e4dd9c056e7322 appengine/standard/firebase/firetactoe/templates/_firebase_config.html